robottelo/host_helpers/contenthost_mixins.py

```diff
@@ -56,6 +56,22 @@ def REPOS(self):
         except KeyError as err:
             raise ValueError(f'Unsupported system version: {self._v_major}') from err
 
+    @cached_property
+    def SATELLITE_CDN_REPOS(self):
+        sat_version = ".".join(settings.server.version.release.split('.')[0:2])
+        return {
+            'satellite': f"satellite-{sat_version}-for-rhel-{self._v_major}-x86_64-rpms",
+            'sat-maintenance': f"satellite-maintenance-{sat_version}-for-rhel-{self._v_major}-x86_64-rpms",
+        }
+
+    @cached_property
+    def CAPSULE_CDN_REPOS(self):
+        sat_version = ".".join(settings.server.version.release.split('.')[0:2])
+        return {
+            'capsule': f"satellite-capsule-{sat_version}-for-rhel-{self._v_major}-x86_64-rpms",
```

Comment:
I'd suggest we make use of robottelo constants here. Probably make the existing one more robust like you have done here, something which need not be hard coded for each version. https://github.com/SatelliteQE/robottelo/blob/fa9126302531728fd602ff7b895f59b93814f567/robottelo/constants/__init__.py#L371

Reply:
To me, constants are static, we need to expand them once a new version of <something> gets released. Contrary to that, what Radek proposes is a dynamic approach.
@jameerpathan111 How are these two cached_properties different from the above methods/properties? Is there something you would like to improve on the above methods as well?

Reply:
Looking at the methods above, I see what you mean. Do you see any danger in keeping what Radek proposes?

Reply:
I don't see any danger, it was just a suggestion.

Reply:
There is already kind of sat_version constant in satellite object, however here we can't use satellite object as we only have a clean RHEL host where we're going to install Satellite. Also there is already a configuration file containing repository ids, but it does not get updated within PIT tests. These are the reasons why I came up with this solution.
Thanks for calling it "robust". We'll see how it works and possibly we could start using it more often. But currently (wthin this PR) I don't want to use it outside of Installer tests.